src/components/task/TaskPanel.tsx

```diff
@@ -0,0 +1,186 @@
+import { Card, CardActions, CardContent, Collapse, ExpansionPanelDetails, ExpansionPanelSummary, Grid, IconButton, Typography } from '@material-ui/core';
+import MuiExpansionPanel from '@material-ui/core/ExpansionPanel';
+import ExpandLessIcon from '@material-ui/icons/ExpandLess';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import { withStyles } from '@material-ui/styles';
+import { APITask, APITest, ConvertToAPITests } from 'evergreen.js/lib/models';
+import * as React from 'react';
+import * as rest from "../../rest/interface";
+import '../../styles.css';
+
+export const StyledExpansionPanel = withStyles({
+  root: {
+    border: '1px solid rgba(0, 0, 0, .125)',
+    boxShadow: "none"
+  }
+})(MuiExpansionPanel);
+
+interface State {
+  hasFailingTests: boolean
+  hasOtherTests: boolean
+  failingTests: APITest[]
+  otherTests: APITest[]
+  isShowingOtherTests: boolean
+}
+
+class Props {
+  public client: rest.Evergreen
+  public task: APITask
+  public status: string
+  public switchTask: (task: APITask) => void
+  public isCurrentTask: boolean
+  public onFinishStateUpdate: () => void;
+}
+
+export class TaskPanel extends React.Component<Props, State> {
+
+  constructor(props: Props) {
+    super(props);
+    this.state = {
+      hasFailingTests: false,
+      hasOtherTests: false,
+      failingTests: [],
+      otherTests: [],
+      isShowingOtherTests: false
+    }
+  }
+
+  public componentDidMount() {
+    this.props.client.getTestsForTask((err, resp, body) => {
+      const tests = ConvertToAPITests(body) as unknown as APITest[];
+      let failingTests: APITest[] = [];
+      let otherTests: APITest[] = [];
+      Array.from(tests).forEach(element => {
```

Comment:
While the vast majority of tests will either be pass or fail, there are potentially 2 others states: https://github.com/evergreen-ci/evergreen/blob/master/globals.go#L76. I think we need a way for someone to discover the actual status of a test

Reply:
I agree with that. Maybe it can be displayed when you hover over the test name, or displayed in a different color? Also, do we consider a skip or a silent failure an actual failure, or would those go in the "other tests" dropdown?

Reply:
They shouldn't be considered failures. A color highlight seems a simple way to denote the status, but I think it would be useful to also say it somewhere. How do you feel about taking either the bottom (or top) of the sidebar to display some metadata about the task?

Reply:
Additional test colors have been added. I can definitely add metadata to the top of the task card. What do you think we should display?